src/components/mainPage/Page.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div class="main-block">
+    <div class="firs-block">
+      <Header />
+    </div>
+    <div class="second-block">
+      <MiddleBlock />
+    </div>
+    <div class="third-block">
+      <ResultList />
+    </div>
+    <div class="footer-block">
+      <div class="footer-logo">
+        <Logo firstPart="netflix" secondPart="roulette" />
```

Comment:
Such data props should be kept in data() option.

Reply:
Fixed